CODEOWNERS

```
@@ -0,0 +1 @@
+* @rbkmoney/erlang @rbkmoney/java @rbkmoney/frontend
```

Я там только, кажется поторопился в fistful-proto. Кажется в будет удобнее, если только на трифтовые файлы требовать ревью от всех.
```suggestion
/proto/ @rbkmoney/erlang @rbkmoney/java @rbkmoney/frontend
/rebar.* @rbkmoney/erlang
/src/ @rbkmoney/erlang
/pom.xml @rbkmoney/java
/package-lock.json @rbkmoney/frontend
/package.json @rbkmoney/frontend
```